src/pipert2/utils/consts/event_format.py

```diff
@@ -0,0 +1,6 @@
+# Event format is { command-optional_flow_name-optional_routine_name }
+EVENT_SEPARATOR = '-'
+EVENT_INDEX = 0
+FLOW_INDEX = 1
+ROUTINE_INDEX = 2
+ROUTINE_CONTAINED_LENGTH = 3
```

Comment:
Try to think of a better name other than `ROUTINE_CONTAINED_LENGTH`?

Reply:
I agree